static/script.js

```diff
@@ -1,97 +1,143 @@
+// solo escucha la informacion de server.ts
+
 async function fetchData() {
-    const response = await fetch("/api/state")
-    const body = await response.json();
-    return body
+  const response = await fetch("/api/state");
+  const body = await response.json();
+  return body;
+}
+
+function classifyStatus(message) {
+  const msg = (message || "").trim().toLowerCase();
+  if (!msg) return { cls: "task-idle", icon: "•" };
+  if (
+    msg.startsWith("acab") ||
+    msg.startsWith("done") ||
+    msg.startsWith("final")
+  )
+    return { cls: "task-done", icon: "✅" };
+  if (
+    msg.startsWith("est") ||
+    msg.startsWith("hac") ||
+    msg.startsWith("tra") ||
+    msg.startsWith("work")
+  )
+    return { cls: "task-active", icon: "⏳" };
+  return { cls: "task-idle", icon: "•" };
```

Comment:
Please put brace around around the if statements because I almost had a kernel panic when seeing two consecutive return statements